frontend/src/modules/Worksheets/components/TextDisplay.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { THEMES, useSettings } from 'design';
+
+export const TextDisplay = ({ text }) => {
+  const { settings } = useSettings();
+
+  const containerStyle = {
+    width: '600px',
+    height: '390px',
+    maxWidth: '100%',
+    margin: '0 auto',
+    padding: '20px',
+    border:
```

Comment:
Instead of using custom colors hardcoded, we could use the available palette (e.g. `backgroundColor: theme.palette.primary.main,` which already make the distinction between dark and light theme. The same applies to the font, instead of hardcoding Arial sans-serif use variable `fontFamily: theme.typography.fontFamily,`